docs/02-app/01-building-your-application/07-configuring/05-mdx.mdx

```diff
@@ -33,7 +33,8 @@ The `@next/mdx` package, and related packages, are used to configure Next.js so
 Install these packages to render MDX with Next.js:
 
 ```bash filename="Terminal"
-npm install @next/mdx @mdx-js/loader @mdx-js/react @types/mdx
+npm install @next/mdx @mdx-js/loader @mdx-js/react
+npm install --save-dev @types/mdx
```

Comment:
I believe one command is more efficient especially when `devDependencies` are not necessary due to our optimized build modes like `output: 'standalone'` which removes the need for a npm install while deploying.